packages/components/src/utils/get-focusable-children.ts

```diff
@@ -25,5 +25,14 @@ const focusDisablingSelector = `:where(${[
 ].join(',')})`;
```

Comment:
To make sure all hidden cases are taken into account:
```suggestion
  '[hidden]:not([hidden="false"])',
].join(',')})`;
```